src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -68,52 +98,190 @@ export const updateTrackStatesAndSave: ActionCreator<
   browserStorageService.saveTrackStates(trackStates);
 };
 
+export const getTracksStateInfo = (
```

Comment:
Confused about the name of this function. "tracksStateInfo" doesn't correspond to what we talked about yesterday

Reply:
✅